create-snowpack-app/app-template-blank-typescript/public/index.html

```diff
@@ -12,7 +12,7 @@
     <img id="img" src="/logo.svg" />
     <canvas id="canvas"></canvas>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <script type="module" src="/dist/index.js"></script>
+    <script type="module" src="/dist/index.ts.js"></script>
```

Comment:
Related question: one of the biggest questions I have when it comes to the pre-build/post-build setup: do you put the source or destination file in your HTML file? What do you do if the filename is hard-to-predict? I’ve gotten a lot of questions from users that don’t know what the final URL will be, and have to run `snowpack build` first before starting the dev server (which means they have to get their app working & buildable first without a dev server…)

Would it just be easier to use the source file here instead? That way the user doesn’t have to know what the final URL will be; Snowpack will just handle it. It may also make it easier for us to do behind-the-scenes build changes such as this extension swap.

Reply:
Yea I like this idea! It meshes well with the rest of our import support which says "users want to import by local file path, and not final built URL". Shouldn't be too hard to scan and find-replace these as a part of build, either. 

Will spin into a prioritized issue

Reply:
Yeah I was thinking more-or-less to scan each HTML import like we already do; if this is a file Snowpack knows is in your source code then replace with destination path.

If it’s not, then leave as-is. In this way, users can still use HTML without being limited by Snowpack (e.g. loading from CDN)